javascript/javascript1/week1/age.js

```diff
@@ -0,0 +1,13 @@
+// a future age calculator
+
+let x = prompt("yearOfBirth");
+let y = prompt("yearFuture");
+let z = y - x;
```

Comment:
Good job. 
Just I recommend you to use `const` not `let` as long as the values will not change.
and to chose names for variables refer to the meaning. Example
```js
const yearOfBirth = prompt("yearOfBirth");
```